### PR TITLE
Fix line endings & add docker build warnings

### PR DESCRIPTION
**Pull request by jamievlin**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -45,6 +45,9 @@ COPY /bot/cogs /rodhaj/bot/cogs/
 COPY /docker/start.sh /rodhaj/start.sh
 COPY /docker/wait-for /rodhaj/wait-for
 
+RUN test -f /rodhaj/bot/.env \
+    || (echo "Environment file not found; please copy .env to bot/.env"; exit 1)
```
Comment (No767) [inline docker/Dockerfile:49]: The idea of having the `.env` file is to only use it during the development lifecycle, and during prod, all we have to do is to pass in a pre-determined `.env` and populate the container with the env. 

Although I did say Windows development is not really supported, it should be fine to develop on as i checked and all wheels are built also for windows. Winloop is used in place of uvloop in order to aid with Windows support. Note that I haven't tested it on Windows, hence the warning

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* text=auto
```
Comment (No767) [inline .gitattributes:1]: Is this auto generated by IntelliJ?

**Review by No767 — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 2 above)
Comment (jamievlin) [reply .gitattributes:1]: Not really, I wrote this myself

File: docker/Dockerfile (hunk 1 above)
Comment (jamievlin) [reply docker/Dockerfile:49]: Wait. Do you want me to remove this line or modify it somehow?

File: .gitattributes (hunk 2 above)
Comment (No767) [reply .gitattributes:1]: Got it

File: docker/Dockerfile (hunk 1 above)
Comment (No767) [reply docker/Dockerfile:49]: Basically in a nutshell, I'm just saying you should remove it

Comment (jamievlin) [reply docker/Dockerfile:49]: aH

Comment (jamievlin) [reply docker/Dockerfile]: Removed the commit

**Review by No767 — APPROVED** [review]
LGTM

**Pull request merged**